### PR TITLE
Remove obsolete CLAPI params

### PR DESCRIPTION
**Pull request by cg-tw**
## Description

Remove obsolete CLAPI params

## Target version (i.e. version that this PR changes)

- [x] 21.10.x (staging)
- [x] 22.04.x (staging)
- [x] 22.10.x (staging)
- [x] 23.04.x (staging)
- [ ] Cloud (staging)
- [ ] Monitoring Connectors (staging)
- [ ] 23.10.x (next)

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs/version-21.10/api/clapi.md [hunk 1]
```diff
@@ -859,22 +860,21 @@ Columns are the following:
...
 ``` shell
-centreon -u admin -p 'centreon' -o CENTBROKERCFG -a addlogger -v "broker cfg for poller test;/var/log/centreon-broker/central-module.log;file"
+centreon -u admin -p 'centreon' -o CENTBROKERCFG -v "broker cfg for poller test;/var/log/centreon-broker/central-module.log;file"
 ```
 
 ``` shell
-centreon -u admin -p 'centreon' -o CENTBROKERCFG -a listlogger -v "broker cfg for poller test"
+centreon -u admin -p 'centreon' -o CENTBROKERCFG -v "broker cfg for poller test"
 id;name
 1;/var/log/centreon-broker/central-module.log
```
Comment (a-launois) [inline i18n/fr/docusaurus-plugin-content-docs/version-21.10/api/clapi.md:879]:
```suggestion
centreon -u admin -p 'centreon' -o CENTBROKERCFG -a listoutput -v "broker cfg for poller test"
id;name
1;Storage
2;RRD
3;PerfData
4:MyFileOutput
```

File: i18n/fr/docusaurus-plugin-content-docs/version-21.10/api/clapi.md [hunk 2]
```diff
@@ -859,22 +860,21 @@ Columns are the following:
...
 
 * **ADDINPUT**
 * **ADDOUTPUT**
-* **ADDLOGGER**
 
 Example:
 
 ``` shell
-centreon -u admin -p 'centreon' -o CENTBROKERCFG -a addlogger -v "broker cfg for poller test;/var/log/centreon-broker/central-module.log;file"
+centreon -u admin -p 'centreon' -o CENTBROKERCFG -v "broker cfg for poller test;/var/log/centreon-broker/central-module.log;file"
```
Comment (a-launois) [inline i18n/fr/docusaurus-plugin-content-docs/version-21.10/api/clapi.md:873]:
```suggestion
centreon -u admin -p 'centreon' -o CENTBROKERCFG -a addoutput -v "broker cfg for poller test;MyFileOutput;file"
```

File: i18n/fr/docusaurus-plugin-content-docs/version-21.10/api/clapi.md [hunk 3]
```diff
@@ -887,34 +887,32 @@ Arguments are composed of the following columns:
...
 
 * **DELINPUT**
 * **DELOUTPUT**
-* **DELLOGGER**
 
 Example:
 
 ``` shell
-centreon -u admin -p 'centreon' -o CENTBROKERCFG -a dellogger -v "broker cfg for poller test;1"
+centreon -u admin -p 'centreon' -o CENTBROKERCFG -v "broker cfg for poller test;1"
```
Comment (a-launois) [inline i18n/fr/docusaurus-plugin-content-docs/version-21.10/api/clapi.md:900]:
```suggestion
centreon -u admin -p 'centreon' -o CENTBROKERCFG -a deloutput -v "broker cfg for poller test;4"
```

File: i18n/fr/docusaurus-plugin-content-docs/version-21.10/api/clapi.md [hunk 4]
```diff
@@ -887,34 +887,32 @@ Arguments are composed of the following columns:
...
 
 * **SETINPUT**
 * **SETOUTPUT**
-* **SETLOGGER**
 
 Example:
 
 ``` shell
-centreon -u admin -p 'centreon' -o CENTBROKERCFG -a setlogger -v "broker cfg for poller test;1;debug;no"
+centreon -u admin -p 'centreon' -o CENTBROKERCFG -v "broker cfg for poller test;1;debug;no"
```
Comment (a-launois) [inline i18n/fr/docusaurus-plugin-content-docs/version-21.10/api/clapi.md:915]:
```suggestion
centreon -u admin -p 'centreon' -o CENTBROKERCFG -a setoutput -v "broker cfg for poller test;1;retry_interval;5'"
```

File: i18n/fr/docusaurus-plugin-content-docs/version-22.04/api/clapi.md [hunk 5]
```diff
@@ -865,16 +866,15 @@ In order to add a new I/O object, use one of the following commands:
 
 * **ADDINPUT**
 * **ADDOUTPUT**
-* **ADDLOGGER**
 
 Example:
 
 ``` shell
-centreon -u admin -p 'centreon' -o CENTBROKERCFG -a addlogger -v "broker cfg for poller test;/var/log/centreon-broker/central-module.log;file"
+centreon -u admin -p 'centreon' -o CENTBROKERCFG -v "broker cfg for poller test;/var/log/centreon-broker/central-module.log;file"
```
Comment (a-launois) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.04/api/clapi.md:873]:
```suggestion
centreon -u admin -p 'centreon' -o CENTBROKERCFG -a addoutput -v "broker cfg for poller test;MyFileOutput;file"
```

File: i18n/fr/docusaurus-plugin-content-docs/version-22.10/api/clapi.md [hunk 6]
```diff
@@ -859,22 +860,21 @@ Columns are the following:
...
 
 * **ADDINPUT**
 * **ADDOUTPUT**
-* **ADDLOGGER**
 
 Example:
 
 ``` shell
-centreon -u admin -p 'centreon' -o CENTBROKERCFG -a addlogger -v "broker cfg for poller test;/var/log/centreon-broker/central-module.log;file"
+centreon -u admin -p 'centreon' -o CENTBROKERCFG -v "broker cfg for poller test;/var/log/centreon-broker/central-module.log;file"
```
Comment (a-launois) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.10/api/clapi.md:873]:
```suggestion
centreon -u admin -p 'centreon' -o CENTBROKERCFG -a addoutput -v "broker cfg for poller test;MyFileOutput;file"
```

File: versioned_docs/version-22.04/api/clapi.md [hunk 7]
```diff
@@ -857,22 +857,21 @@ Columns are the following:
...
 
 * **ADDINPUT**
 * **ADDOUTPUT**
-* **ADDLOGGER**
 
 Example:
 
 ``` shell
-centreon -u admin -p 'centreon' -o CENTBROKERCFG -a addlogger -v "broker cfg for poller test;/var/log/centreon-broker/central-module.log;file"
+centreon -u admin -p 'centreon' -o CENTBROKERCFG -v "broker cfg for poller test;/var/log/centreon-broker/central-module.log;file"
```
Comment (a-launois) [inline versioned_docs/version-22.04/api/clapi.md:870]:
```suggestion
centreon -u admin -p 'centreon' -o CENTBROKERCFG -a addoutput -v "broker cfg for poller test;MyFileOutput;file"
```

File: versioned_docs/version-23.04/api/clapi.md [hunk 8]
```diff
@@ -857,22 +858,21 @@ Columns are the following:
...
 
 * **ADDINPUT**
 * **ADDOUTPUT**
-* **ADDLOGGER**
 
 Example:
 
 ``` shell
-centreon -u admin -p 'centreon' -o CENTBROKERCFG -a addlogger -v "broker cfg for poller test;/var/log/centreon-broker/central-module.log;file"
+centreon -u admin -p 'centreon' -o CENTBROKERCFG -v "broker cfg for poller test;/var/log/centreon-broker/central-module.log;file"
```
Comment (a-launois) [inline versioned_docs/version-23.04/api/clapi.md:871]:
```suggestion
centreon -u admin -p 'centreon' -o CENTBROKERCFG -a addoutput -v "broker cfg for poller test;MyFileOutput;file"
```

File: i18n/fr/docusaurus-plugin-content-docs/version-22.04/api/clapi.md [hunk 9]
```diff
@@ -865,16 +866,15 @@ In order to add a new I/O object, use one of the following commands:
...
 ```
 
 ``` shell
-centreon -u admin -p 'centreon' -o CENTBROKERCFG -a listlogger -v "broker cfg for poller test"
+centreon -u admin -p 'centreon' -o CENTBROKERCFG -v "broker cfg for poller test"
 id;name
 1;/var/log/centreon-broker/central-module.log
```
Comment (a-launois) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.04/api/clapi.md:879]:
```suggestion
centreon -u admin -p 'centreon' -o CENTBROKERCFG -a listoutput -v "broker cfg for poller test"
id;name
1;Storage
2;RRD
3;PerfData
4:MyFileOutput
```

File: i18n/fr/docusaurus-plugin-content-docs/version-22.10/api/clapi.md [hunk 10]
```diff
@@ -859,22 +860,21 @@ Columns are the following:
...
 ```
 
 ``` shell
-centreon -u admin -p 'centreon' -o CENTBROKERCFG -a listlogger -v "broker cfg for poller test"
+centreon -u admin -p 'centreon' -o CENTBROKERCFG -v "broker cfg for poller test"
 id;name
 1;/var/log/centreon-broker/central-module.log
```
Comment (a-launois) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.10/api/clapi.md:879]:
```suggestion
centreon -u admin -p 'centreon' -o CENTBROKERCFG -a listoutput -v "broker cfg for poller test"
id;name
1;Storage
2;RRD
3;PerfData
4:MyFileOutput
```

File: i18n/fr/docusaurus-plugin-content-docs/version-23.04/api/clapi.md [hunk 11]
```diff
@@ -859,22 +860,21 @@ Columns are the following:
...
 ``` shell
-centreon -u admin -p 'centreon' -o CENTBROKERCFG -a addlogger -v "broker cfg for poller test;/var/log/centreon-broker/central-module.log;file"
+centreon -u admin -p 'centreon' -o CENTBROKERCFG -v "broker cfg for poller test;/var/log/centreon-broker/central-module.log;file"
 ```
 
 ``` shell
-centreon -u admin -p 'centreon' -o CENTBROKERCFG -a listlogger -v "broker cfg for poller test"
+centreon -u admin -p 'centreon' -o CENTBROKERCFG -v "broker cfg for poller test"
 id;name
 1;/var/log/centreon-broker/central-module.log
```
Comment (a-launois) [inline i18n/fr/docusaurus-plugin-content-docs/version-23.04/api/clapi.md:879]:
```suggestion
centreon -u admin -p 'centreon' -o CENTBROKERCFG -a listoutput -v "broker cfg for poller test"
id;name
1;Storage
2;RRD
3;PerfData
4:MyFileOutput
```

File: versioned_docs/version-21.10/api/clapi.md [hunk 12]
```diff
@@ -857,22 +857,21 @@ Columns are the following:
...
 ``` shell
-centreon -u admin -p 'centreon' -o CENTBROKERCFG -a addlogger -v "broker cfg for poller test;/var/log/centreon-broker/central-module.log;file"
+centreon -u admin -p 'centreon' -o CENTBROKERCFG -v "broker cfg for poller test;/var/log/centreon-broker/central-module.log;file"
 ```
 
 ``` shell
-centreon -u admin -p 'centreon' -o CENTBROKERCFG -a listlogger -v "broker cfg for poller test"
+centreon -u admin -p 'centreon' -o CENTBROKERCFG -v "broker cfg for poller test"
 id;name
 1;/var/log/centreon-broker/central-module.log
```
Comment (a-launois) [inline versioned_docs/version-21.10/api/clapi.md:876]:
```suggestion
centreon -u admin -p 'centreon' -o CENTBROKERCFG -a listoutput -v "broker cfg for poller test"
id;name
1;Storage
2;RRD
3;PerfData
4:MyFileOutput
```

File: versioned_docs/version-22.04/api/clapi.md [hunk 13]
```diff
@@ -857,22 +857,21 @@ Columns are the following:
...
 ```
 
 ``` shell
-centreon -u admin -p 'centreon' -o CENTBROKERCFG -a listlogger -v "broker cfg for poller test"
+centreon -u admin -p 'centreon' -o CENTBROKERCFG -v "broker cfg for poller test"
 id;name
 1;/var/log/centreon-broker/central-module.log
```
Comment (a-launois) [inline versioned_docs/version-22.04/api/clapi.md:876]:
```suggestion
centreon -u admin -p 'centreon' -o CENTBROKERCFG -a listoutput -v "broker cfg for poller test"
id;name
1;Storage
2;RRD
3;PerfData
4:MyFileOutput
```

File: versioned_docs/version-22.10/api/clapi.md [hunk 14]
```diff
@@ -857,22 +858,21 @@ Columns are the following:
...
 ``` shell
-centreon -u admin -p 'centreon' -o CENTBROKERCFG -a addlogger -v "broker cfg for poller test;/var/log/centreon-broker/central-module.log;file"
+centreon -u admin -p 'centreon' -o CENTBROKERCFG -v "broker cfg for poller test;/var/log/centreon-broker/central-module.log;file"
 ```
 
 ``` shell
-centreon -u admin -p 'centreon' -o CENTBROKERCFG -a listlogger -v "broker cfg for poller test"
+centreon -u admin -p 'centreon' -o CENTBROKERCFG -v "broker cfg for poller test"
 id;name
 1;/var/log/centreon-broker/central-module.log
```
Comment (a-launois) [inline versioned_docs/version-22.10/api/clapi.md:877]:
```suggestion
centreon -u admin -p 'centreon' -o CENTBROKERCFG -a listoutput -v "broker cfg for poller test"
id;name
1;Storage
2;RRD
3;PerfData
4:MyFileOutput
```

File: versioned_docs/version-23.04/api/clapi.md [hunk 15]
```diff
@@ -857,22 +858,21 @@ Columns are the following:
...
 ```
 
 ``` shell
-centreon -u admin -p 'centreon' -o CENTBROKERCFG -a listlogger -v "broker cfg for poller test"
+centreon -u admin -p 'centreon' -o CENTBROKERCFG -v "broker cfg for poller test"
 id;name
 1;/var/log/centreon-broker/central-module.log
```
Comment (a-launois) [inline versioned_docs/version-23.04/api/clapi.md:877]:
```suggestion
centreon -u admin -p 'centreon' -o CENTBROKERCFG -a listoutput -v "broker cfg for poller test"
id;name
1;Storage
2;RRD
3;PerfData
4:MyFileOutput
```

File: i18n/fr/docusaurus-plugin-content-docs/version-22.10/api/clapi.md [hunk 16]
```diff
@@ -887,34 +887,32 @@ Arguments are composed of the following columns:
...
 
 * **DELINPUT**
 * **DELOUTPUT**
-* **DELLOGGER**
 
 Example:
 
 ``` shell
-centreon -u admin -p 'centreon' -o CENTBROKERCFG -a dellogger -v "broker cfg for poller test;1"
+centreon -u admin -p 'centreon' -o CENTBROKERCFG -v "broker cfg for poller test;1"
```
Comment (a-launois) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.10/api/clapi.md:900]:
```suggestion
centreon -u admin -p 'centreon' -o CENTBROKERCFG -a deloutput -v "broker cfg for poller test;4"
```

File: versioned_docs/version-22.04/api/clapi.md [hunk 17]
```diff
@@ -885,34 +884,32 @@ Arguments are composed of the following columns:
...
 
 * **DELINPUT**
 * **DELOUTPUT**
-* **DELLOGGER**
 
 Example:
 
 ``` shell
-centreon -u admin -p 'centreon' -o CENTBROKERCFG -a dellogger -v "broker cfg for poller test;1"
+centreon -u admin -p 'centreon' -o CENTBROKERCFG -v "broker cfg for poller test;1"
```
Comment (a-launois) [inline versioned_docs/version-22.04/api/clapi.md:897]:
```suggestion
centreon -u admin -p 'centreon' -o CENTBROKERCFG -a deloutput -v "broker cfg for poller test;4"
```

File: versioned_docs/version-22.10/api/clapi.md [hunk 18]
```diff
@@ -885,34 +885,32 @@ Arguments are composed of the following columns:
...
 
 * **DELINPUT**
 * **DELOUTPUT**
-* **DELLOGGER**
 
 Example:
 
 ``` shell
-centreon -u admin -p 'centreon' -o CENTBROKERCFG -a dellogger -v "broker cfg for poller test;1"
+centreon -u admin -p 'centreon' -o CENTBROKERCFG -v "broker cfg for poller test;1"
```
Comment (a-launois) [inline versioned_docs/version-22.10/api/clapi.md:898]:
```suggestion
centreon -u admin -p 'centreon' -o CENTBROKERCFG -a deloutput -v "broker cfg for poller test;4"
```

File: versioned_docs/version-23.04/api/clapi.md [hunk 19]
```diff
@@ -885,34 +885,32 @@ Arguments are composed of the following columns:
...
 
 * **DELINPUT**
 * **DELOUTPUT**
-* **DELLOGGER**
 
 Example:
 
 ``` shell
-centreon -u admin -p 'centreon' -o CENTBROKERCFG -a dellogger -v "broker cfg for poller test;1"
+centreon -u admin -p 'centreon' -o CENTBROKERCFG -v "broker cfg for poller test;1"
```
Comment (a-launois) [inline versioned_docs/version-23.04/api/clapi.md:898]:
```suggestion
centreon -u admin -p 'centreon' -o CENTBROKERCFG -a deloutput -v "broker cfg for poller test;4"
```

File: i18n/fr/docusaurus-plugin-content-docs/version-22.04/api/clapi.md [hunk 20]
```diff
@@ -887,34 +887,32 @@ Arguments are composed of the following columns:
...
 
 * **SETINPUT**
 * **SETOUTPUT**
-* **SETLOGGER**
 
 Example:
 
 ``` shell
-centreon -u admin -p 'centreon' -o CENTBROKERCFG -a setlogger -v "broker cfg for poller test;1;debug;no"
+centreon -u admin -p 'centreon' -o CENTBROKERCFG -v "broker cfg for poller test;1;debug;no"
```
Comment (a-launois) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.04/api/clapi.md:915]:
```suggestion
centreon -u admin -p 'centreon' -o CENTBROKERCFG -a setoutput -v "broker cfg for poller test;1;retry_interval;5'"
```

File: i18n/fr/docusaurus-plugin-content-docs/version-22.10/api/clapi.md [hunk 21]
```diff
@@ -887,34 +887,32 @@ Arguments are composed of the following columns:
...
 
 * **SETINPUT**
 * **SETOUTPUT**
-* **SETLOGGER**
 
 Example:
 
 ``` shell
-centreon -u admin -p 'centreon' -o CENTBROKERCFG -a setlogger -v "broker cfg for poller test;1;debug;no"
+centreon -u admin -p 'centreon' -o CENTBROKERCFG -v "broker cfg for poller test;1;debug;no"
```
Comment (a-launois) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.10/api/clapi.md:915]:
```suggestion
centreon -u admin -p 'centreon' -o CENTBROKERCFG -a setoutput -v "broker cfg for poller test;1;retry_interval;5'"
```

File: i18n/fr/docusaurus-plugin-content-docs/version-23.04/api/clapi.md [hunk 22]
```diff
@@ -903,18 +902,17 @@ centreon -u admin -p 'centreon' -o CENTBROKERCFG -a dellogger -v "broker cfg for
...
 
 * **SETINPUT**
 * **SETOUTPUT**
-* **SETLOGGER**
 
 Example:
 
 ``` shell
-centreon -u admin -p 'centreon' -o CENTBROKERCFG -a setlogger -v "broker cfg for poller test;1;debug;no"
+centreon -u admin -p 'centreon' -o CENTBROKERCFG -v "broker cfg for poller test;1;debug;no"
```
Comment (a-launois) [inline i18n/fr/docusaurus-plugin-content-docs/version-23.04/api/clapi.md:915]:
```suggestion
centreon -u admin -p 'centreon' -o CENTBROKERCFG -a setoutput -v "broker cfg for poller test;1;retry_interval;5'"
```

File: versioned_docs/version-21.10/api/clapi.md [hunk 23]
```diff
@@ -885,34 +884,32 @@ Arguments are composed of the following columns:
...
 
 * **SETINPUT**
 * **SETOUTPUT**
-* **SETLOGGER**
 
 Example:
 
 ``` shell
-centreon -u admin -p 'centreon' -o CENTBROKERCFG -a setlogger -v "broker cfg for poller test;1;debug;no"
+centreon -u admin -p 'centreon' -o CENTBROKERCFG -v "broker cfg for poller test;1;debug;no"
```
Comment (a-launois) [inline versioned_docs/version-21.10/api/clapi.md:912]:
```suggestion
centreon -u admin -p 'centreon' -o CENTBROKERCFG -a setoutput -v "broker cfg for poller test;1;retry_interval;5'"
```

File: versioned_docs/version-22.04/api/clapi.md [hunk 24]
```diff
@@ -885,34 +884,32 @@ Arguments are composed of the following columns:
...
 
 * **SETINPUT**
 * **SETOUTPUT**
-* **SETLOGGER**
 
 Example:
 
 ``` shell
-centreon -u admin -p 'centreon' -o CENTBROKERCFG -a setlogger -v "broker cfg for poller test;1;debug;no"
+centreon -u admin -p 'centreon' -o CENTBROKERCFG -v "broker cfg for poller test;1;debug;no"
```
Comment (a-launois) [inline versioned_docs/version-22.04/api/clapi.md:912]:
```suggestion
centreon -u admin -p 'centreon' -o CENTBROKERCFG -a setoutput -v "broker cfg for poller test;1;retry_interval;5'"
```

File: versioned_docs/version-22.10/api/clapi.md [hunk 25]
```diff
@@ -885,34 +885,32 @@ Arguments are composed of the following columns:
...
 
 * **SETINPUT**
 * **SETOUTPUT**
-* **SETLOGGER**
 
 Example:
 
 ``` shell
-centreon -u admin -p 'centreon' -o CENTBROKERCFG -a setlogger -v "broker cfg for poller test;1;debug;no"
+centreon -u admin -p 'centreon' -o CENTBROKERCFG -v "broker cfg for poller test;1;debug;no"
```
Comment (a-launois) [inline versioned_docs/version-22.10/api/clapi.md:913]:
```suggestion
centreon -u admin -p 'centreon' -o CENTBROKERCFG -a setoutput -v "broker cfg for poller test;1;retry_interval;5'"
```

File: i18n/fr/docusaurus-plugin-content-docs/version-23.04/api/clapi.md [hunk 26]
```diff
@@ -887,13 +887,12 @@ Arguments are composed of the following columns:
...
-#### Delinput, Deloutput and Dellogger
+#### Delinput and Deloutput
 
 In order to remove an I/O object from the Centreon Broker configuration, use one of the following commands:
 
 * **DELINPUT**
 * **DELOUTPUT**
-* **DELLOGGER**
 
 Example:
```
Comment (a-launois) [inline i18n/fr/docusaurus-plugin-content-docs/version-23.04/api/clapi.md:898]: Missing example update:
`centreon -u admin -p 'centreon' -o CENTBROKERCFG -a deloutput -v "broker cfg for poller test;4"`

**Review by a-launois — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs/version-21.10/api/clapi.md [hunk 27]
```diff
@@ -859,22 +860,21 @@ Columns are the following:
 | 1      | Parameter key of the I/O   |
 | 2      | Parameter value of the I/O |
 
-#### Addinput, Addoutput and Addlogger
+#### Addinputa and Addoutput
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-21.10/api/clapi.md:863]:
```suggestion
#### Addinput and Addoutput
```

**Review by a-launois — APPROVED** [review]

**Pull request merged**